src/toplc.ml

```diff
@@ -656,17 +651,92 @@ let get_ancestors h c =
   ga c;
   U.hashtbl_fold_keys (fun c cs -> c :: cs) cs []
 
-let name_of_class c =
-  B.Utils.UTF8.to_string (B.Name.external_utf8_for_class c)
-
-let mk_full_method_name c mn =
-  name_of_class c ^ "." ^ mn
-
 let get_overrides h c m =
   let ancestors = get_ancestors h c in
   let qualify c =  mk_full_method_name c m.method_name in
   (List.map qualify ancestors, m.method_arity)
 
+let rec pp_for_parameter_list = function
```

Comment:
Please use `Format.pp_print_list`. (Also, note that the implementation here is quadratic because of the string concatenation.)

Reply:
Changed to `Format.pp_print_list` added `pp_parameter` function